File: EventService/src/main/java/com/lambda/EventService/Models/Api/EventWrapperDTO.java
package com.lambda.EventService.Models.Api;

import com.lambda.EventService.Models.Entity.EnuEventStatus;
import com.lambda.EventService.Models.Entity.Event;
import com.lambda.EventService.Models.Entity.EventType;
import com.lambda.EventService.Models.Entity.Location;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class EventWrapperDTO {
    public Event event;
    public Location location;
    public EnuEventStatus enuEventStatus;
    public EventType eventType;
    public Long userId;
}

File: Events-Frontend/Events/src/app/components/events/models/DTO/EventWrapperDTO.ts
import { event } from '../interfaces/event';
import { enuEventStatus } from '../interfaces/enuEventStatus';
import { eventType } from '../interfaces/eventType';
import { location } from '../interfaces/location';



export interface EventWrapperDTO {
    event:event;
    location:location;
    enuEventStatus:enuEventStatus;
    eventType:eventType;
    userId:number;
}